attributes/default.rb

```diff
@@ -96,6 +96,14 @@
     default['cfncluster']['dcv']['gl'] = "nice-dcv-gl_2019.1.544-1_amd64.ubuntu1804.deb"  # required to enable GPU sharing
     default['cfncluster']['dcv']['sha256sum'] = "41a0b4112c435b3a57de3ae46cfe8cdf90c33a216f488e3bfab944f034f6067f"
   end
+when 'amazon'
```

Comment:
since the following definition are the same used for centos7, can we try to put them in common?
Also, given that we define
```
default['cfncluster']['dcv']['supported_os'] = %w[centos7 ubuntu18 amazon2]
```
we could also double check if ```if node['platform_version'].to_i```  is necessary or could be omitted and just leave the switch case for platform

Reply:
Yeah i didn't handle the conditionals together because the case statement keys on the platform family without respect to the version. But you are right, it should be done that way. I'll change the case statement to key on the same <platform family><platform version int> combo used in the supported_os list.